### PR TITLE
trivial: fixup setup helpers to install right version of meson

### PR DESCRIPTION
**Pull request by superm1**
if meson wasn't installed by pip it won't have a version string

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: contrib/ci/fwupd_setup_helpers.py [hunk 1]
```diff
@@ -58,9 +58,12 @@ def test_markdown(debug):
...
 
 def test_meson(debug):
-    from importlib.metadata import version
+    from importlib.metadata import version, PackageNotFoundError
 
-    new_enough = version("meson") >= MINIMUM_MESON
+    try:
+        new_enough = version("meson") >= MINIMUM_MESON
+    except PackageNotFoundError:
+        new_enough = False
```
Comment (hughsie) [inline contrib/ci/fwupd_setup_helpers.py:66]: Could we call into the system meson and get the version? It seems a bit antisocial to just ignore the distro version.

Comment (superm1) [reply contrib/ci/fwupd_setup_helpers.py:66]: I agree but I'm not sure how to. Any ideas?

Comment (hughsie) [reply contrib/ci/fwupd_setup_helpers.py:66]: Would `meson --version` work?

Comment (superm1) [reply contrib/ci/fwupd_setup_helpers.py]: Yeah I think I got that setup now, take a look.

**Review by hughsie — APPROVED** [review]

**Pull request merged**